src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements Drawable, AreaCalculable {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. It seems that 'Drawable' and 'AreaCalculable' interfaces are not conceptually similar and should not be used together in this way.